src/utils/_forms.scss

```diff
@@ -29,3 +29,84 @@
     display: none;
   }
 }
+
+// Any form field: input, textarea, ...
+// e.g: @include formField(input);
+
+@mixin form-field($field) {
+  overflow: hidden;
+
+  &-#{$field} {
+    border-color: $c-gray-light;
+    border-style: solid;
+    border-width: 1px;
+    font-size: $fz-base;
+    padding-left: $p-l;
+    padding-right: $p-l;
+    width: 100%;
+
+    &:not(:disabled)::placeholder {
+      color: $c-gray-dark;
+    }
+
+    &:disabled {
+      background: $c-gray-lightest;
+
+      &::placeholder {
+        color: $c-gray-light;
+      }
+    }
+
+    &:focus {
+      box-shadow: inset 0 0 3px 0 $c-primary;
+    }
+  }
+
+  //sui-AtomInput-success & sui-AtomInput-error
+  $validation: success $c-success, error $c-error;
+
+  @each $type, $color in $validation {
+    &-#{$type} {
+      label {
+        color: $color;
+      }
+
+      #{$field} {
+        border-color: $color;
+      }
+
+      &-validation-text {
+        color: $color;
+        display: block;
+        font-size: $fz-xs;
+        margin: $m-s 0 0;
+      }
+    }
+  }
+}
+
+// Help text for form elements
+@mixin help-text {
+  color: $c-gray-light;
+  display: block;
+  font-size: $fz-xs;
+  margin: $m-s 0 0;
+}
+
+// Size for form elements
+@mixin field-size($field) {
+  $size: null !default;
+  @if($field == 'input') {
+    $size: medium 40px, small 30px;
+  } @else if($field == 'textarea') {
+    $size: medium 400px, small 200px;
+  }
+
+  @each $type, $h in $size {
+    &-#{$type} {
+      #{$field} {
+        height: $h;
+      }
```

Comment:
This ouput something like:

```scss
.sui-Form-medium input {
  height: 40px;
}
.sui-Form-small input {
  height: 30px;
}
```
we usually avoid style elements. what do you think about adding a class to the styled element?

Reply:
I agree, but in this case, we can use in different types of elements form, and the name of the class is defined in the react component